test/instrumentation/github-issue-2715.test.js

```diff
@@ -0,0 +1,52 @@
+const agent = require('../..').start({
+  serviceName: 'test-composite-context',
+  captureExceptions: false,
+  metricsInterval: 0,
+  centralConfig: false,
+  cloudProvider: 'none',
+  spanCompressionEnabled: true
+})
+
+const mockClient = require('../_mock_http_client')
+
+const tape = require('tape')
+tape.test(function (suite) {
+  suite.test(function (t) {
```

Comment:
nits:
- Why the wrap in a suite here for a single test?
- Both the `tape.test(` and `suite.test(` should get a test name to assist with self-documenting and correlation with output. Currently it looks like:

```
% node test/instrumentation/github-issue-2715.test.js
TAP version 13
# (anonymous)
# (anonymous)
ok 1 should be strictly equal
ok 2 should be strictly equal
ok 3 should be strictly equal
ok 4 should be strictly equal


{"log.level":"error","@timestamp":"2022-05-24T17:58:52.688Z","log":{"logger":"elastic-apm-node"},"ecs":{"version":"1.6.0"},"message":"APM Server transport error: error fetching APM Server version: connect ECONNREFUSED 127.0.0.1:8200"}

1..4
# tests 4
# pass  4
```